related_products_ecommerce/tests/test_cart.py

```diff
@@ -0,0 +1,93 @@
+from odoo.tests.common import TransactionCase
+from odoo.tests import tagged
+from odoo import Command
+
+
+@tagged("post_install", "-at_install")
+class TestCartRelatedProducts(TransactionCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.p1_template = cls.env['product.template'].create({
+            'name': 'P1'
```

Comment:
```suggestion
            'name': 'Product A'
```
name can be given better